docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -311,81 +313,17 @@ Possible parameters are:
-
-    - `get_individual_FD` determines whether or not to output the data for individual fundamental diagram in the given measurement area, which is based on the Voronoi density $$\rho_i$$,  velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon of each pedestrian $$i$$ in a given measurement area but not mean value over space. If true, the related data will be written in the folder `./Output/Fundamental_Diagram/IndividualFD/` in the output file `IFD_J_`.
-
-- `one_dimensional` should be used when pedestrians move on a line
-  [single-file experiment](http://www.asim.uni-wuppertal.de/datenbank/own-experiments/corridor/1d-single-file-no-2.html).
+- since version X.Y.Z: `global_IFD` indicates a global measurement area encompassing the entire geometry for which individual data (IFD) are calculated. This parameter is set to `false`by default. 
 
-- `cut_by_circle` determines whether to cut each cell by circle or not. Two options `radius` of the circle and the number of `edges` have to be supplied for approximating the circle if `enabled` is *true*.
+- since version X.Y.Z: `use_blind_points` allows to calculate Voronoi cells in measurement areas even if less than four pedestrians are present. This is realized with the help of blind points are automatically defined outside the geometry. This parameter is set to `true`by default.
 
-- `profiles` indicates whether to calculate the profiles over time and space. If `enabled` is true, the resolution which is decided by the parameters `grid_size_x` and `grid_size_x` should be set.
+- since version X.Y.Z: `vel_calculation` indicates the approach that is used for the velocity calculation. By default the `Voronoi` approach is chosen but it can be changed to `Arithmetic` if needed. See [Method D](jpsreport_method_D.html) for details. 
 
 Possible output data are:
-  - `/Fundamental_Diagram/Classical_Voronoi/`: output file `rho_v_Voronoi_J_` with mean density and velocity of over time (frame, $$rho(t)$$, $$v(t)$$).
-  - `/Fundamental_Diagram/IndividualFD/`: output file `IFD_J_` with Voronoi density $$\rho_i$$, velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon of each pedestrian $$i$$.
+  - `/Fundamental_Diagram/Classical_Voronoi/`: output file `rho_v_Voronoi_[velocity_calculation_type]_[filename.txt]_id_[local_measurement_area_id].dat` with mean density and velocity over time (frame, $$rho(t)$$, $$v(t)$$). 
+  - `/Fundamental_Diagram/IndividualFD/`: output file `IFD_rho_v_Voronoi_[velocity_calculation_type]_[filename.txt]_id_[local_measurement_area_id].dat` with Voronoi density $$\rho_i$$, velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon of each pedestrian $$i$$ in measurement area.
```

Comment:
After a first application I noticed that the output file for individual data is unnecessarily complicated. The following names would be sufficient:
- `IFD_local_[filename.txt]_id_[local_measurement_area_id].dat`
- `IFD_global_[filename.txt]_id_[local_measurement_area_id].dat`

The output file for `<local_IFD enabled="true"/>` contains the instantaneous velocity $$v_{i}(t)$$, the individual density $$\rho_{i}(t)$$ (based on Voronoi polygon), position ($$x_{i}$$, $$y_{i}$$, $$z_{i}$$), the Voronoi polygon and the intersection (with the measurement area) of each pedestrian $$i$$ in a given measurement area.

The output file for `<global_IFD enabled="true"/>` contains the instantaneous velocity $$v_{i}(t)$$, the individual density $$\rho_{i}(t)$$ (based on Voronoi polygon), position ($$x_{i}$$, $$y_{i}$$, $$z_{i}$$) and the Voronoi polygon of each pedestrian $$i$$.

Reply:
@JetteSchumann We should talk about the function `Method_D::OpenFileIndividual()` ([Method_D.cpp, line 280](https://github.com/JuPedSim/jpscore/blob/aaf58552edc86bb09c2f7d146f1efa2dca5d7a9b/jpsreport/methods/Method_D.cpp#L280)) next week.

Reply:
Done with PR #844